test/HelloOBJ/HelloOBJ.cpp

```diff
@@ -0,0 +1,43 @@
+#include "RCube/Core/Graphics/Effects/GammaCorrectionEffect.h"
+#include "RCube/Core/Graphics/MeshGen/Obj.h"
+#include "RCubeViewer/RCubeViewer.h"
+
+int main()
+{
+    using namespace rcube;
+
+    // Properties to configure the viewer
+    viewer::RCubeViewerProps props;
+    props.resolution = glm::vec2(1280 /*4096*/, 720/*2160*/); // 720p
+    props.MSAA = 2; // turn on 2x multisampling
+
+    //Create a viewer
+    viewer::RCubeViewer viewer(props);
+
+    //Load obj file 
+    std::string input_obj_file = std::string(OBJ_RESOURCE_PATH) + "/" +                                 
+                                 "armadillo.obj";
+    MeshData mesh = rcube::loadOBJ(input_obj_file);
+
+    //make sure the mesh is OK    
+    //assert(mesh.valid());
+
+    //Add the loaded obj to the viewer
+    EntityHandle mesh_handle = viewer.addOBJ("OBJMesh", mesh);
```

Comment:
See my previous comment above, this line could be `EntityHandle mesh_handle = viewer.addSurface("OBJMesh", mesh);`